### PR TITLE
Add kotlin test cases for lint module

### PR DESCRIPTION
**Pull request by hotchemi**
## Issue

resolves https://github.com/permissions-dispatcher/PermissionsDispatcher/issues/373

## Overview

- Add kotlin test cases for lint module.
  - Tweak CallOnRequestPermissionsResultDetector to address onRequestPermissionsResult extension function.

### CODE REVIEW

File: lint/src/main/java/permissions/dispatcher/CallOnRequestPermissionsResultDetector.java [hunk 1]
```diff
@@ -55,11 +57,15 @@ public void visitClass(UClass node) {
...
 
+        private final boolean isKotlin;
+
         private boolean hasRuntimePermissionAnnotation;
 
+
         private OnRequestPermissionsResultChecker(JavaContext context, UClass klass) {
             this.context = context;
             this.className = klass.getName();
+            isKotlin = context.getPsiFile() != null && "kotlin".equals(context.getPsiFile().getLanguage().getID());
```
[the remark this answers is not in the document]
Comment (shiraji) [reply lint/src/main/java/permissions/dispatcher/CallOnRequestPermissionsResultDetector.java]: That's good to know.

**Comment by hotchemi** [comment]
Can I merge?

File: lint/src/main/java/permissions/dispatcher/CallOnRequestPermissionsResultDetector.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (mannodermaus) [reply lint/src/main/java/permissions/dispatcher/CallOnRequestPermissionsResultDetector.java]: Is `getLanguage()` an enum that already contains a Kotlin member? If so, and we have access to that, might as well use that

[the remark this answers is not in the document]
Comment (hotchemi) [reply lint/src/main/java/permissions/dispatcher/CallOnRequestPermissionsResultDetector.java]: Thank you! Actually `Language` is not an Enum but just an abstract class.. 😅A bit unsmart but we gotta check ID 🤔 
https://github.com/JetBrains/intellij-community/blob/571fbe96754e6c762fe8f7f8532ae2665bfbae61/platform/core-api/src/com/intellij/lang/Language.java#L42

**Comment by shiraji** [comment]
Sure!

File: lint/src/main/java/permissions/dispatcher/CallOnRequestPermissionsResultDetector.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (mannodermaus) [reply lint/src/main/java/permissions/dispatcher/CallOnRequestPermissionsResultDetector.java]: Ok, it's OK then!

**Comment by hotchemi** [comment]
Thank you guys! I'll release next version tonight!

**Pull request merged**